packages/application-generic/src/services/in-memory-provider/in-memory-provider.service.ts

```diff
@@ -121,43 +133,60 @@ export class InMemoryProviderService {
     }
   }
 
-  private isElasticacheEnabled(): boolean {
-    return (
-      !!process.env.ELASTICACHE_CLUSTER_SERVICE_HOST &&
-      !!process.env.ELASTICACHE_CLUSTER_SERVICE_PORT
-    );
-  }
-
-  private getClientAndConfigForCluster(): {
+  public getClientAndConfigForCluster(providerId: InMemoryProviderEnum): {
```

Comment:
Wanted to unit test it so changed the privacy scope. Can be temporary.

Reply:
btw you can hack it in the tests, like have this one still private and in the tests access it like this:
```
(obj as any).functionName();
```
the "private" scope is a TS thing, but the code transpiles to JS at the end ;)

Reply:
I am sorry but that's not the best advice as TS already has the bracket notation accessor as safe escape while keeping the typing. https://github.com/microsoft/TypeScript/issues/19335
At the end I moved in a different PR this method as generic function in a private Node.js module, so that settles my discomfort about this change. 😅